src/posthog-core.ts

```diff
@@ -110,6 +110,7 @@ const defaultConfig = (): PostHogConfig => ({
     cookie_name: '',
     loaded: __NOOP,
     store_google: true,
+    custom_utm_params: [],
```

Comment:
Should be `custom_campaign_params`? (I suggest "campaign" since that's more generic than "UTM")

Reply:
Yep sounds reasonable.